readme.md

```diff
@@ -1,6 +1,8 @@
-# ezXSS
+# ezXSS docker-compose
```

Comment:
I would still like ezXSS to be working without docker. Could you make this so that it would work both ways?